labgrid/remote/client.py

```diff
@@ -1708,8 +1708,35 @@ def __str__(self):
         return self.value
 
 
-def get_parser() -> argparse.ArgumentParser:
-    parser = argparse.ArgumentParser()
+def get_parser(include_undocumented=False) -> argparse.ArgumentParser:
```

Comment:
`include_undocumented` does a lot more than including undocumented things. This needs a better name.